gruntfile.js

```diff
@@ -3,26 +3,33 @@ var cssnano = require('cssnano');
 
 module.exports = function (grunt) {
 
-  grunt.loadNpmTasks('grunt-contrib-concat');
-  grunt.loadNpmTasks('gruntify-eslint');
-  grunt.loadNpmTasks('grunt-contrib-uglify');
-  grunt.loadNpmTasks('grunt-contrib-clean');
-  grunt.loadNpmTasks('grunt-contrib-copy');
-  grunt.loadNpmTasks('grunt-contrib-watch');
-  grunt.loadNpmTasks('grunt-html2js');
-  grunt.loadNpmTasks('grunt-shell');
-  grunt.loadNpmTasks('grunt-if');
-  grunt.loadNpmTasks('grunt-filerev');
-  grunt.loadNpmTasks('grunt-usemin');
-  grunt.loadNpmTasks('grunt-replace');
-  grunt.loadNpmTasks('grunt-config');
-  grunt.loadNpmTasks('grunt-postcss');
+  var NpmTasks = [
+    'grunt-contrib-concat',
+    'gruntify-eslint',
+    'grunt-contrib-uglify',
+    'grunt-contrib-clean',
+    'grunt-contrib-copy',
+    'grunt-contrib-watch',
+    'grunt-html2js',
+    'grunt-shell',
+    'grunt-filerev',
+    'grunt-usemin',
+    'grunt-replace',
+    'grunt-config',
+    'grunt-postcss'
+  ];
+  for (var n in NpmTasks)  {
+    if (vendor[NpmTasks].hasOwnProperty(n)) {
+      grunt.loadNpmTasks(NpmTasks[n]);
+    }
+  }
```

Comment:
Sure, if you have npm and node installed you need to run `npm i -D load-grunt-tasks` and it should appear in your "devDependencies" section of `package.json`. Otherwise, you can add `"load-grunt-tasks": "^3.5.2"` entry to "devDependencies".